Misc/NEWS.d/next/Library/2025-03-12-11-53-32.gh-issue-130940.81K1Tg.rst

```diff
@@ -0,0 +1,4 @@
+The ``PyConfig.use_system_logger`` attribute, introduced in Python 3.12.2, has
```

Comment:
This change is for Python 3.13. Do you mean Python 3.13.2?

Reply:
🤦 I did.